src/bci_build/package/appcontainers.py

```diff
@@ -484,3 +484,39 @@ def _get_nginx_kwargs(os_version: OsVersion):
     )
     for os_version in (OsVersion.TUMBLEWEED,)
 ]
+
+_REDIS_ENTRYPOINT = (Path(__file__).parent / "redis" / "entrypoint.tar.gz").read_bytes()
```

Comment:
please don't add binary blobs here. if this is coming from somewhere, please fetch it from https://github.com/SUSE/BCI-dockerfile-generator/blob/main/update-files.sh and add it unpacked.

Reply:
Removed the entrypoint. Blob is now gone.